app/server/logging/jsondb/src/main/java/io/syndesis/server/logging/jsondb/controller/PodLogMonitor.java

```diff
@@ -66,13 +66,13 @@ public PodLogMonitor(ActivityTrackingController logsController, Pod pod) throws
             throw new IOException("Could not determine the pod name");
         }
 
-        Map<String, String> annotations = pod.getMetadata().getAnnotations();
-        this.integrationId = annotations.get(OpenShiftService.INTEGRATION_ID_ANNOTATION);
+        Map<String, String> labels = pod.getMetadata().getLabels();
+        this.integrationId = labels.get(OpenShiftService.INTEGRATION_ID_ANNOTATION);
         if (this.integrationId == null) {
             throw new IOException("Could not determine the integration id that is being run on the pod: " + this.podName);
         }
 
-        this.deploymentVersion = annotations.get(OpenShiftService.DEPLOYMENT_VERSION_ANNOTATION);
+        this.deploymentVersion = labels.get(OpenShiftService.DEPLOYMENT_VERSION_ANNOTATION);
```

Comment:
same here. Constant is name `ANNOTATION`, but used for a label

Reply:
Actually I would remove the annotation constants in favor of the label constants.